data/test/scenarios/wml_tests/ScenarioWML/EventWML/ActionWML/DirectActionsWML/modify_unit_recall_cost.cfg

```diff
@@ -0,0 +1,83 @@
+#textdomain wesnoth-test
+# Actions:
+# Create Charlie, belonging to side 2, on the recall list
+# Increase Charlie's recall cost 

```

Comment:
The CI check is complaining about two things in this file: it has CRLF line endings, and some lines have trailing whitespace.

Running `data/tools/wmlindent data/test/scenarios/wml_tests/ScenarioWML/EventWML/ActionWML/DirectActionsWML/modify_unit_recall_cost.cfg` will fix both issues.